tests/config.c

```diff
@@ -51,14 +54,17 @@ config_set_defaults(master_config *cfg)
       .allocator_capacity       = GiB_TO_B(TEST_CONFIG_DEFAULT_DISK_SIZE_GB),
       .cache_capacity           = GiB_TO_B(TEST_CONFIG_DEFAULT_CACHE_SIZE_GB),
       .btree_rough_count_height = 1,
-      .filter_remainder_size    = 6,
+      .filter_remainder_size    = 9,
```

Comment:
This is just going to trigger the autoconfig which sets it to 4. 9 is actually going to larger in practice because filters very rarely cover an entire node.